subsys/net/lib/lwm2m/Kconfig

```diff
@@ -47,7 +47,7 @@ config LWM2M_COAP_BLOCK_SIZE
 	range 64 1024
 	help
 	  CoAP block size used by LWM2M when performing block-wise
-	  transfers. Possible values: 16, 32, 64, 128, 256, 512 and 1024.
```

Comment:
Why not 16 and 32? These are still valid values for block size, I don't think we need to impose any limits on the LwM2M side.

Reply:
I decided to go with modifying the _help_ instead of the _range_ itself. I shall do it the other way round.

Reply:
Ah, now I noticed the the range did not correspond to the help string. Nevermind then, please ignore this comment.